File: mobile/src/main/kotlin/de/scout/fireplace/ui/AbstractAnimatorListener.java
package de.scout.fireplace.ui;

import android.animation.Animator;

public abstract class AbstractAnimatorListener implements Animator.AnimatorListener {

  @Override
  public void onAnimationStart(Animator animation) {
    /* no op */
  }

  @Override
  public void onAnimationEnd(Animator animation) {
    /* no op */
  }

  @Override
  public void onAnimationCancel(Animator animation) {
    /* no op */
  }

  @Override
  public void onAnimationRepeat(Animator animation) {
    /* no op */
  }
}

File: feature/src/main/kotlin/de/scout/fireplace/feature/ui/AbstractAnimatorListener.kt
package de.scout.fireplace.feature.ui

import android.animation.Animator

abstract class AbstractAnimatorListener : Animator.AnimatorListener {

  override fun onAnimationStart(animation: Animator) = Unit

  override fun onAnimationEnd(animation: Animator) = Unit

  override fun onAnimationCancel(animation: Animator) = Unit

  override fun onAnimationRepeat(animation: Animator) = Unit
}
